nodes/core/core/58-debug.html

```diff
@@ -225,6 +229,11 @@ <h3>Details</h3>
             delete RED._debug;
         },
         oneditprepare: function() {
+            if (typeof this.console === "string") {
```

Comment:
Can we double check the logic around types here? Above you set the default values of these properties as booleans. Here you only act if they are strings.

Reply:
yeah - let me re-visit.

Reply:
so yes this was for in "old" scheme it was a string - so when opened for first time it checks and forces it to now be a boolean - So I think it's what I intended.